app/jobs/remove_old_throttles_job.rb

```diff
@@ -0,0 +1,43 @@
+class RemoveOldThrottlesJob < ApplicationJob
+  queue_as :low
+
+  WINDOW = 30.days.freeze
+
+  include GoodJob::ActiveJobExtensions::Concurrency
+
+  good_job_control_concurrency_with(
+    enqueue_limit: 1,
+    perform_limit: 1,
+    key: -> do
+      rounded = TimeService.round_time(time: arguments.first, interval: 1.hour)
+      "remove-old-throttles-#{rounded.to_i}"
+    end,
+  )
+
+  discard_on GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError
+
+  def perform(now, limit: 1000, total_limit: 100_000)
+    max_window = Throttle::THROTTLE_CONFIG.map { |_, config| config[:attempt_window] }.max
```

Comment:
Just to check my understanding, the idea with this is to delete 30 days after a throttle has expired, and grabbing the max window guarantees we'd not drop any prematurely?

Reply:
Yup! I figured if we ever add a super long throttle, this makes sure we don't delete it prematurely